dfastmi/batch/ConfigurationChecker.py

```diff
@@ -37,393 +37,114 @@
 import dfastmi.kernel.legacy
```

Comment:
why is kernel.legacy imported here? is it used here or can it be removed?

Reply:
updated, seems to be rebasing problem